Content.Server/Singularity/EntitySystems/GravityWellSystem.cs

```diff
@@ -61,201 +58,226 @@ public override void Update(float frameTime)
     /// </summary>
-    /// <param name="mapPos">The epicenter of the gravity pulse.</param>
-    /// <param name="maxRange">The maximum distance at which entities can be affected by the gravity pulse.</param>
-    /// <param name="minRange">The minimum distance at which entities can be affected by the gravity pulse. Exists to prevent div/0 errors.</param>
-    /// <param name="baseRadialDeltaV">The base amount of velocity that will be added to entities in range towards the epicenter of the pulse.</param>
-    /// <param name="baseTangentialDeltaV">The base amount of velocity that will be added to entities in range counterclockwise relative to the epicenter of the pulse.</param>
-    public void GravPulse(MapCoordinates mapPos, float maxRange, float minRange = 0.0f, float baseRadialDeltaV = 0.0f, float baseTangentialDeltaV = 0.0f)
-        => GravPulse(mapPos, maxRange, minRange, new Matrix3(
-            baseRadialDeltaV, +baseTangentialDeltaV, 0.0f,
-            -baseTangentialDeltaV, baseRadialDeltaV, 0.0f,
-            0.0f, 0.0f, 1.0f
-        ));
+    /// <param name="uid">The uid of the gravity well to set the scan period for.</param>
+    /// <param name="value">The new scan period for the gravity well.</param>
+    /// <param name="gravWell">The state of the gravity well to set the scan period for.</param>
+    public void SetTangentialAcceleration(EntityUid uid, float value, GravityWellComponent? gravWell = null)
+    {
+        if(!Resolve(uid, ref gravWell))
+            return;
 
-    #endregion GravPulse
+        if (MathHelper.CloseTo(gravWell.BaseTangentialAcceleration, value))
+            return;
 
-    #region Getters/Setters
+        gravWell.BaseTangentialAcceleration = value;
+        UpdateMatrix(uid, gravWell);
+    }
 
     /// <summary>
-    /// Sets the pulse period for a gravity well.
-    /// If the new pulse period implies that the gravity well was intended to pulse already it does so immediately.
+    /// Sets the scan period for a gravity well.
+    /// If the new scan period implies that the gravity well was intended to scan already it does so immediately.
     /// </summary>
-    /// <param name="uid">The uid of the gravity well to set the pulse period for.</param>
-    /// <param name="value">The new pulse period for the gravity well.</param>
-    /// <param name="gravWell">The state of the gravity well to set the pulse period for.</param>
-    public void SetPulsePeriod(EntityUid uid, TimeSpan value, GravityWellComponent? gravWell = null)
+    /// <param name="uid">The uid of the gravity well to set the scan period for.</param>
+    /// <param name="value">The new scan period for the gravity well.</param>
+    /// <param name="gravWell">The state of the gravity well to set the scan period for.</param>
+    public void SetScanPeriod(EntityUid uid, TimeSpan value, GravityWellComponent? gravWell = null)
```

Comment:
Does this even need to be settable.

Reply:
It exists, an admin might want to modify it for <insert reason here>, and doing so will not endanger players so probably.